k-distribution/tests/regression-new/map-symbolic-tests-haskell/inkeys-6-spec.k.out

```diff
@@ -0,0 +1,23 @@
+  #Not ( {
```

Comment:
If I'm not mistaken, we already discussed that we can't support this at the moment, so it will go to `failing`.